UnitTestExample/Presenter.swift

```diff
@@ -8,16 +8,18 @@
 
 import Foundation
 
-protocol ViewProtocol : class{
-    func dataIsReady(collection: [ArtPiece]?)
+protocol PresenterInterface {
+    func updateView()
```

Comment:
can we change their position always variables at top functions at bottom as standard guideline